language/move-prover/src/bytecode_translator.rs

```diff
@@ -767,7 +922,11 @@ impl<'env> ModuleTranslator<'env> {
     /// This generates boogie code for everything after the function signature
     /// The function body is only generated for the `FunctionEntryPoint::Definition`
     /// version of the function.
-    fn generate_inline_function_body(&self, func_target: &FunctionTarget<'_>) {
+    fn generate_inline_function_body(
+        &self,
+        func_target: &FunctionTarget<'_>,
+        rewritten_bytecode: &Option<Vec<Bytecode>>,
```

Comment:
I don't understand why we need a bytecode parameter here. The idea of the transformation pipeline is that the rewritten code is what you get from function_target. This seems to be not matching the current design.